app/src/main/java/com/quran/labs/androidquran/ui/PagerActivity.java

```diff
@@ -3,6 +3,7 @@
 import android.annotation.TargetApi;
 import android.app.ProgressDialog;
 import android.app.SearchManager;
+import android.appwidget.AppWidgetManager;
```

Comment:
Unused import? It looks like there is no significant change in this file at all. Please revert.